pkg/api/controller.go

```diff
@@ -3600,3 +3634,10 @@ func (c *Controller) authorize(w http.ResponseWriter, r *http.Request, perms per
 	}
 	return true
 }
+
+func (c *Controller) isNameValid(name string, nameType string) (bool, string) {
```

Comment:
Is the second arg intentionally unused?

Reply:
Unused? Not sure I follow.
`nameType` is used in the string formatting.